### PR TITLE
CDS simplified

### PR DESCRIPTION
**Pull request by TomGoBravo**
* Change `update_covid_data_scraper.py` to do its own fetch, use the project Li `locationID` and remove cleanup logic not needed any longer
* factor out the helper functions shared with other update scripts to `update_helpers`

### CODE REVIEW

**Comment by ghop02** [comment]
One small note - it's probably better to just do the data updating through the normal data update action on master to reduce merge conflicts and make sure there aren't any artifacts from intermediate code

File: scripts/update_cmdc.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     CommonFields,
     COMMON_FIELDS_TIMESERIES_KEYS,
 )
-from scripts.update_covid_data_scraper import FieldNameAndCommonField, load_county_fips_data
+from scripts.update_helpers import FieldNameAndCommonField, load_county_fips_data, rename_fields
```
[the remark this answers is not in the document]
Comment (TomGoBravo) [reply scripts/update_cmdc.py:23]: Fixed in https://github.com/covid-projections/covid-data-public/pull/95

File: scripts/update_cmdc.py [hunk 2]
```diff
@@ -97,13 +97,13 @@ class Config:
 
     @staticmethod
     def make_with_data_root(
-        data_root: pathlib.Path, cmdc_key: Optional[str] = None
+        data_root: pathlib.Path, cmdc_key: Optional[str], log,
```
[the remark this answers is not in the document]
Comment (TomGoBravo) [reply scripts/update_cmdc.py:100]: Fixed in https://github.com/covid-projections/covid-data-public/pull/95

File: scripts/update_covid_data_scraper.py [hunk 3]
```diff
@@ -1,217 +1,181 @@
...
-        data[CommonFields.COUNTRY] = "USA"
-        data[CommonFields.STATE] = data[Fields.STATE].apply(
-            lambda x: US_STATE_ABBREV[x] if x in US_STATE_ABBREV else x
+        df_us_mask = df[Fields.COUNTRY] == "United States"
+
+        df_counties = df[df_us_mask & (df[Fields.LEVEL] == "county")].copy()
+        # Using str.slice instead of str.extract is really ugly but is easier than fixing the DataFrame assignment
+        # errors I had with str.extract.
+        bad_location_id = ~df_counties[Fields.LOCATION_ID].str.match(
+            r"\Aiso1:us#iso2:us-..#fips:\d{5}\Z"
```
[the remark this answers is not in the document]
Comment (TomGoBravo) [reply scripts/update_covid_data_scraper.py:94]: No strong reason. I remember when I was learning REs thinking \A \Z makes more sense than ^ and $, though perhaps the lack of ubiquity is a big downside of them.

File: scripts/update_covid_data_scraper.py [hunk 4]
```diff
@@ -1,217 +1,181 @@
...
-                "Removing columns not in CommonFields", columns=col_not_in_fields_or_common
+                "Removing duplicate timeseries points",
+                duplicates=df.loc[
+                    duplicates_mask, [Fields.LOCATION_ID, CommonFields.FIPS, Fields.DATE]
+                ],
             )
+            df = df.loc[~duplicates_mask, :]
 
-        return data
+        df[CommonFields.CASES] = pd.to_numeric(df[Fields.CASES])
```
[the remark this answers is not in the document]
Comment (TomGoBravo) [reply scripts/update_covid_data_scraper.py:150]: Added comment in https://github.com/covid-projections/covid-data-public/pull/95

**Pull request merged**